test/e2e/framework/helm/helm.go

```diff
@@ -37,10 +37,23 @@ func Install(input InstallInput) {
 	os.Chdir("../..")
 	defer os.Chdir(cwd)
 
+	chartDir := input.Config.HelmChartDir
+
+	//resolve helm dependency
+	dependencyArgs := append([]string{
+		"dependency",
+		"update",
+		chartDir,
+		fmt.Sprintf("--namespace=%s", framework.NamespaceKubeSystem),
+		"--debug",
+	})
+	err = helm(dependencyArgs)
```

Comment:
Why are we running this? We already do `--dependency-update` as part of the helm install command. 

https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/5d1596aeb8d15aad8fb3c81826f978ecca2aa4c6/test/e2e/framework/helm/helm.go#L50

Reply:
removed